### ISSUE TITLE
Rename prompt always assumes you want the earliest-opened tab that's visible

### ISSUE THREAD

**Issue by Shadow-Hog** [sirjuddington/SLADE#1121]
**SLADE Version**: 3.1.11 (x86 build)
**Operating System**: Windows 10 Pro 64-bit

**Issue Details**:
Do the following:

1. Open two or more archives. 
2. Drag/snap the tabs for each (save for one, doesn't matter which) to any side of the screen you want so that you can view them all at once in separate divisions.
3. Left-click any entry in the first archive you opened that you want, doesn't matter.
4. Right-click on any entry in any other archive that was opened, and select Rename.

**Expected:** The prompt will ask me what I want to rename the entry selected in Step 4's archive to.
**Actual:** The prompt will ask me what I want to rename the entry selected in Step 3's archive to. It doesn't matter which tab you click, it will always assume you're only acting on the very first archive opened.

Experimenting with it a little further, it appears it goes for whichever tab *currently displayed* was opened first. In the above scenario, since all archives are visible at the same time, that'd always be the very first archive opened, but it's a bit more complex than that. So, say you have archive №1 and archive №3 sharing a division to the left, and archive №2 and archive №4 sharing a division to the right. For all four combinations of which tabs are visible:

- №1 and №2 are visible: SLADE opts for №1.
- №3 and №2 are visible: SLADE opts for №2.
- №1 and №4 are visible: SLADE opts for №1.
- №3 and №4 are visible: SLADE opts for №3.

This does mean that when you're _not_ using divisions, the rename prompt works exactly as expected; as there are no divisions of the screen, only one archive is ever visible at a time, so it would be the earliest visible archive opened by default. It does, however, make doing renaming work when comparing against another archive a bit more annoying, as you either have to constantly hop from one tab to another, or else be certain you opened the archives in the correct order before proceeding, lest you end up renaming the entries in the reference archive instead of the target archive.

**Comment by Gaerzi** [comment]
> 2. Drag/snap the tabs for each (save for one, doesn't matter which) to any side of the screen you want so that you can view them all at once in separate divisions.

I didn't even know you could do that... Pretty cool but yes, I can see how it's gonna wreck havoc with the method SLADE uses to determine which tab is active.

**Comment by sirjuddington** [comment]
Yeah that's one feature of wxAUI that I didn't disable in case anyone wanted to use it, but that was a very long time ago and I had since forgotten it was even a thing :P Most everything is written with the assumption that only one tab is visible at once, and I'm not sure how much work it'd take to properly support tab splitting
